src/v2/Components/Inquiry/Views/InquiryLogin.tsx

```diff
@@ -51,10 +57,15 @@ export const InquiryLogin: React.FC = () => {
     try {
       const { user } = await login({ email: inquiry.email!, ...state })
 
+      const relayEnvironment = createRelaySSREnvironment({ user })
```

Comment:
Why is this necessary? (Code comment here would be good since we don't have to do this anywhere else in the app)